commentHandler.go

```diff
@@ -308,3 +362,19 @@ func checkTransition(requestedAction string, currentState string) (string, bool)
 
 	return "", false
 }
+
+func removeMilestone(client *github.Client, ctx context.Context, URL string) error {
+	req, err := client.NewRequest("PATCH", URL, &struct {
+		Milestone interface{} `json:"milestone"`
+	}{})
+	if err != nil {
+		return err
+	}
+
+	_, err = client.Do(ctx, req, nil)
```

Comment:
can be changed to
`if _, err = client.Do(ctx, req, nil); err != nil {`

Reply:
Sounds reasonable ^